ocaml/idl/datamodel.ml

```diff
@@ -3442,6 +3442,10 @@ let vdi_resize_online = call
     ~name:"resize_online"
     ~in_oss_since:None
     ~in_product_since:rel_rio
+    ~lifecycle: [
+      Published, rel_rio, "";
+      Removed, rel_inverness, "Remove feature of online VDI resize because it is not supported by any of the storage backends."
```

Comment:
I think we should phrase this a bit differently: `"Remove online VDI resize feature because ..."`. @lindig any opinions about how to best phrase this?

Reply:
That's an useful suggestion from native speaker, thank you. ;-)

Reply:
Since it is obvious that we removed an API call, I would simply state the reason: `Online VDI resize is not supported by any of the storage backends.`

Reply:
Updated per your comments.